libs/design/src/atoms/form/quantity-field/quantity-input/quantity-input.component.ts

```diff
@@ -0,0 +1,116 @@
+import { coerceNumberProperty } from '@angular/cdk/coercion';
+import {
+  Component,
+  Input,
+  ViewChild,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  OnInit,
+  OnDestroy,
+} from '@angular/core';
+import {
+  FormControl,
+  NgControl,
+} from '@angular/forms';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { DaffInputComponent } from '../../input/public_api';
+
+@Component({
+  selector: 'daff-quantity-input',
+  templateUrl: './quantity-input.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DaffQuantityInputComponent implements OnInit, OnDestroy {
+  @ViewChild(DaffInputComponent) input: DaffInputComponent;
+
+  /**
+   * @docs
+   * The minimum number for the quantity input field
+   */
+  @Input() min = 1;
+
+  /**
+   * @docs
+   * The maximum number for the quantity input field
+   */
+  @Input() max = 10;
+
+  /**
+   * A new control for the nested native input.
+   * We don't bind the native input directly to the inherited form control
+   * to avoid triggering updates on the input event.
+   * Instead, we listen for the change event and manually patch form control values.
+   */
+  _inputControl = new FormControl();
+  // TODO: should this not use this.input as the source of truth for focused?
+  focused = false;
```

Comment:
Seems like this should just be a getter from `this.input.focused` like the select component does.